cloudformation/cloudformation.yaml

```diff
@@ -368,11 +373,6 @@ Resources:
       Environment:
         Variables:
           'Stage': !Sub ${Stage}
-  checkerLambdaLogGroup:
-    Type: "AWS::Logs::LogGroup"
-    Properties:
-      LogGroupName: !Sub "/aws/lambda/${checkerLambda}"
-    DependsOn: checkerLambda
```

Comment:
For some reason If i didn't create the lambda log group explicitly here I could not create the stack since it looks like the log group was not created until the lambda logs for the first time.
However, this makes updates fail since it tries to create the log group every time and fails with an error saying there is already a log group with that name.
So I guess the best we can do is have something that will work for updating the stack and if we ever want to create it from scratch we can do it in a parts, first create the lambda, then the metrics